src/mpid/ch4/netmod/ofi/ofi_progress.h

```diff
@@ -23,23 +23,26 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_NM_progress(int vni, int blocking)
 {
     int mpi_errno;
     struct fi_cq_tagged_entry wc[MPIDI_OFI_NUM_CQ_ENTRIES];
```

Comment:
Please notice that defining `MPIDI_OFI_NUM_CQ_ENTRIES` as CVAR turns `wc` into a variable length array. Compiler will produce additional instructions here in this case.

It is better to leave constant length array here by allocating something like `wc[MPIDI_OFI_MAX_NUM_CQ_ENTRIES]`, where `MPIDI_OFI_MAX_NUM_CQ_ENTRIES` is maximum value of `MPIDI_OFI_NUM_CQ_ENTRIES`.